tests/operator_test.go

```diff
@@ -153,6 +153,23 @@ var _ = Describe("Tests needing the restore of nodes", func() {
 			Expect(err).ToNot(HaveOccurred())
 		}
 
+		Eventually(func() bool {
+			services, err := f.K8sClient.CoreV1().Services(f.CdiInstallNs).List(context.TODO(), metav1.ListOptions{})
+			Expect(err).ToNot(HaveOccurred(), "failed getting CDI services")
+			for _, service := range services.Items {
+				if service.Name != "cdi-prometheus-metrics" {
```

Comment:
why check != cdi-prometheus-metrics?

Reply:
Because that one never has any endpoints, so checking for endpoints there will always fail. It only gets endpoints for certain specific tests.